global/prometheus-alertmanager/templates/config.yaml

```diff
@@ -128,72 +150,94 @@ data:
           severity: critical
           region: admin|ap-ae-1|ap-au-1|ap-cn-1|ap-jp-1|ap-jp-2|ap-sa-1|eu-de-1|eu-de-2|eu-nl-1|eu-ru-1|la-br-1|na-ca-1|na-us-1|na-us-3
 
-      - receiver: slack_net_warning
+      - receiver: slack_net_critical
         continue: true
         match_re:
           tier: net
-          severity: warning
+          severity: critical
           region: admin|ap-ae-1|ap-au-1|ap-cn-1|ap-jp-1|ap-jp-2|ap-sa-1|eu-de-1|eu-de-2|eu-nl-1|eu-ru-1|la-br-1|na-ca-1|na-us-1|na-us-3
 
-      - receiver: slack_net_critical
+      - receiver: slack_net_warning
         continue: true
         match_re:
           tier: net
-          severity: critical
+          severity: warning
           region: admin|ap-ae-1|ap-au-1|ap-cn-1|ap-jp-1|ap-jp-2|ap-sa-1|eu-de-1|eu-de-2|eu-nl-1|eu-ru-1|la-br-1|na-ca-1|na-us-1|na-us-3
 
-      # ----  Just Notification Slack Channels ----
-      - receiver: slack_os_critical
+      - receiver: slack_net_info
         continue: true
         match_re:
-          tier: os
-          severity: critical
+          tier: net
+          severity: info
           region: admin|ap-ae-1|ap-au-1|ap-cn-1|ap-jp-1|ap-jp-2|ap-sa-1|eu-de-1|eu-de-2|eu-nl-1|eu-ru-1|la-br-1|na-ca-1|na-us-1|na-us-3
 
-      - receiver: slack_k8s_critical
+      # ---- QA / DEV & Labs ----
+      - receiver: slack_qa
         continue: true
         match_re:
-          tier: k8s
-          severity: critical
-          cluster_type: controlplane
-          region: admin|ap-ae-1|ap-au-1|ap-cn-1|ap-jp-1|ap-jp-2|ap-sa-1|eu-de-1|eu-de-2|eu-nl-1|eu-ru-1|la-br-1|na-ca-1|na-us-1|na-us-3
+          severity: critical | warning | info
+          region: qa-de-1
 
-      - receiver: slack_kks_critical
+      - receiver: slack_dev
         continue: true
         match_re:
-          tier: kks
-          severity: critical
-          region: admin|ap-ae-1|ap-au-1|ap-cn-1|ap-jp-1|ap-jp-2|ap-sa-1|eu-de-1|eu-de-2|eu-nl-1|eu-ru-1|la-br-1|na-ca-1|na-us-1|na-us-3
+          severity: critical | warning | info
+          region: staging | lab-1
 
-      - receiver: slack_by_tier_and_severity
+      # ---- special notification channels ----
+      # owner: Thomas Grainchen
+      - receiver: slack_nannies
         continue: true
         match_re:
-          tier: os|kks|metal|net
-          severity: info|warning
-          region: admin|qa-de-1|ap-ae-1|ap-au-1|ap-cn-1|ap-jp-1|ap-jp-2|ap-sa-1|eu-de-1|eu-de-2|eu-nl-1|eu-ru-1|la-br-1|na-ca-1|na-us-1|na-us-3
+          context: nanny 
```

Comment:
The `context` label is used to inhibit alerts. I'm not really sure if it's wise to use it for routing as well.

Reply:
me neither .. that's also why I thought it could be use-full for dev purposes to route to an own channel and content is owner driven .. Martin V. & I want do review the label soon .. ask for your input/experience/opinion (undoubtedly sure :))